bindata/network/ovn-kubernetes/managed/single-zone-interconnect/004-config.yaml

```diff
@@ -0,0 +1,162 @@
+---
```

Comment:
**Note**: the only difference between this file and the one in multi-zone in managed is that it lacks the configuration for libovsdb logs.